Shamaz/Controllers/PlayersTaskViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SecondViewController.swift
+//  Shamaz
+//
+//  Created by Alexandra Ivanova on 31/03/2020.
+//  Copyright © 2020 Alex Ivanova. All rights reserved.
+//
+
+import UIKit
+
+class PlayersTaskViewController: UIViewController {
+
+    @IBOutlet weak var nextPlayerButton: UIButton!
+    @IBOutlet weak var playerTaskLabel: UILabel!
+    
+    var receivedPhrase = "Player's task"
+    var nextPlayerPhrase = ""
+    var nextPlayerNumber = Int.random(in: 1...10)
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        nextPlayerButton.layer.cornerRadius = Design.buttonRadius
+        nextPlayerButton.backgroundColor = .white
+        nextPlayerButton.setTitleColor(Design.primaryThemeColor, for: [])
+        nextPlayerButton.setTitle("Who's NeXT?", for: [])
```

Comment:
Typo, Maybe?

Reply:
Who's NeXT?
I got that from the project description actually: "Add a third button for the next player nomination: Ex."Who's NeXT?""
😜

Reply:
Ok 😊